components/searchable_user_list/searchable_user_list.jsx

```diff
@@ -237,7 +239,10 @@ export default class SearchableUserList extends React.Component {
                 <div className='filter-row'>
                     {filterRow}
                     <div className='col-sm-12'>
-                        <span className='member-count pull-left'>{this.renderCount(usersToDisplay)}</span>
+                        <span
+                            id='searchableUserListTotal'
+                            className='member-count pull-left'
+                        >{this.renderCount(usersToDisplay)}</span>
```

Comment:
Not sure why this is not failing in ESLint but we usually do something like,
```javascript
<span
    id='searchableUserListTotal'
    className='member-count pull-left'
>
    {this.renderCount(usersToDisplay)}
</span>
```

Reply:
I included this change in the today's PR.